docs/providers/kubernetes.md

```diff
@@ -41,12 +43,21 @@ k8s_store = ff.register_k8s(
     team="featureform-team"
 )
 ```
+
 {% endcode %}
 
+### Updatable Configuration Fields
+
+* `description`
+* `docker_image`
+* File store updatable configuration fields
```

Comment:
Should this be deleted? "* File store updatable configuration fields"

Reply:
I intended this to mean, "Whichever fields are mutable in your preferred file store"